src/detectors/BEMC/ProtoCluster_factory_EcalBarrelImagingProtoClusters.h

```diff
@@ -29,6 +29,7 @@ class ProtoCluster_factory_EcalBarrelImagingProtoClusters : public JFactoryT<edm
     void Init() override{
         auto app = GetApplication();
 
+        app->SetDefaultParameter("BEMC:EcalBarrelImagingProtoClusters:input_tag", m_input_tag, "Name of input collection to use");
         app->SetDefaultParameter("BEMC:EcalBarrelImagingProtoClusters::neighbourLayersRange",    m_neighbourLayersRange);
```

Comment:
missing other initialization?

Reply:
Good catch. The initialization was being done after calling SetDefaultParameter. Fixing this.

Reply:
what about inits of u_localDistXY, u_layerDistEtaPhi. m_minClusterEdep, m_minClusterNhits?

Reply:
Those are initialized in the generic algorithm class here:
[https://github.com/eic/EICrecon/blob/a0fc59705820d099fa8f24f7b1f69d0da6cccad1/src/algorithms/calorimetry/ImagingTopoCluster.h#L68]()

You are right though that some of them are set in the ATHENA configuration script here:

[https://eicweb.phy.anl.gov/EIC/benchmarks/physics_benchmarks/-/blob/master/options/reconstruction.py#L593]()

I will fix this.

Reply:
OK. I've push a commit that copies the values from the juggler configuration script to overwrite the defaults from the generic algorithm. This should be in alignment with what juggler was doing.